### DISCUSSION TITLE
Dynamic enumvalues

### DISCUSSION THREAD

**Discussion by pbtaffi** [firecmsco/firecms#178] [Q&A] [answered]
Is it possible to provide dynamic enumValues ? One of my field enumValues is a dynamic list which can grow or shrink. How can I ensure that enumValues shows the updated list. Thanks.

**Comment by fgatti675** [comment, marked as answer]
Converting this issue in a discussion since it is more appropriate.
When defining your navigation, you can use a `NavigationBuilder` which can be `async` so you can create dynamic schemas and properties: https://firecms.co/docs/navigation
